crates/forge/bin/cmd/fmt.rs

```diff
@@ -39,12 +40,15 @@ pub struct FmtArgs {
     /// In 'check' and stdin modes, outputs raw formatted code instead of the diff.
     #[arg(long, short)]
     raw: bool,
+
+    #[command(flatten)]
+    pub watch: WatchArgs,
 }
 
 impl_figment_convert_basic!(FmtArgs);
 
 impl FmtArgs {
-    pub fn run(self) -> Result<()> {
+    pub async fn run(self) -> Result<()> {
```

Comment:
this didn't change, right?

so we can keep this non async?

Reply:
Fixed!